misk-policy/src/test/kotlin/misk/policy/opa/RealOpaPolicyEngineTest.kt

```diff
@@ -78,6 +78,23 @@ internal class RealOpaPolicyEngineTest {
     assertThat(evaluate).isEqualTo(BasicResponse("a"))
     assertThat(requestCaptor.value).isEqualTo("{\"input\":{\"someValue\":1}}")
   }
+  @Test
```

Comment:
nit: spacing (add an empty line after closing `}`

Reply:
This is good attention to detail, thank you. :-)